identity/issuer/issuer.go

```diff
@@ -600,6 +602,27 @@ func generateExistenceMTProof(mt *merkletree.MerkleTree, hi, root *merkletree.Ha
 	return mtp, nil
 }
 
+// entryInTree checks if a given entry is in the merkle tree startin from the
+// rootKey.  If rootKey is nil, the current merkle tree root is used.
+func entryInTree(mt *merkletree.MerkleTree, rootKey *merkletree.Hash, entry *merkletree.Entry) error {
```

Comment:
maybe this function can be a utility in `merkletree` package instead of `issuer`? (as this can be used by other parts of the repository)

Reply:
Makes sense!